src/lib/crud.route.factory.ts

```diff
@@ -67,24 +67,13 @@ export class CrudRouteFactory {
     } = {
         tableName: '',
     };
-    private paginationType: PaginationType;
 
     constructor(
         protected target: any,
         protected crudOptions: CrudOptions,
     ) {
         this.entityInformation(crudOptions.entity);
 
-        const paginationType = crudOptions.routes?.readMany?.paginationType ?? CRUD_POLICY[Method.READ_MANY].default.paginationType;
-        const isPaginationType = (
-            <TEnum extends Record<string, unknown>>(enumType: TEnum) =>
-            (nextCursor: unknown): nextCursor is TEnum[keyof TEnum] =>
-                Object.values(enumType).includes(nextCursor as TEnum[keyof TEnum])
-        )(PaginationType);
-        if (!isPaginationType(paginationType)) {
-            throw new TypeError(`invalid PaginationType ${paginationType}`);
-        }
-        this.paginationType = paginationType;
```

Comment:
Previously, `this.paginationType` was assigned a value in the constructor, and it reflected only the config value for readMany route. `this.paginationType` was only referenced in the generation logic for swagger docs.